Gopkg.toml

```diff
@@ -20,7 +20,27 @@
 #  name = "github.com/x/y"
 #  version = "2.4.0"
 
+required = [
+  "k8s.io/code-generator/cmd/client-gen",
+  "k8s.io/code-generator/cmd/conversion-gen",
+  "k8s.io/code-generator/cmd/deepcopy-gen",
+  "k8s.io/code-generator/cmd/defaulter-gen",
+  "k8s.io/code-generator/cmd/informer-gen",
+  "k8s.io/code-generator/cmd/lister-gen",
+]
```

Comment:
❓ Why list all the commands rather than the parent package?
```
required  = [
    "k8s.io/code-generator",
]
```

Reply:
We don't require the entire parent package, we require just the dependencies of the actual binaries themselves. If we depend on the parent package, we'll pull in a lot more than is necessary.

Shortly I'm going to follow up with a PR after running `dep prune`, that removes a *lot* of unused packages. If I required the parent, it would not remove some of those despite them not being used 😄